distribution/tools/keystore-cli/src/main/java/org/elasticsearch/common/settings/cli/ShowKeyStoreCommand.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.common.settings.cli;
```

Comment:
I put this in the `.setting.cli` package to deal with the split packages check.
I can move it back to `.settings` instead, and update the exceptions list, but I think our preference is to use a new package.

However, I couldn't put the test into the `setting.cli` package, because it needs to access package protected methods on `KeyStoreWrapper`.

Reply:
We should add a `TestKeyStoreWrapper` that inherits KeyStoreWrapper and makes those public for tests to call. It could be in a followup though.